src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void draw(Figure figure);

```

Comment:
The method 'draw' in the interface 'Drawable' should not take a 'Figure' as a parameter. Each figure should know how to draw itself. The method should be parameterless, and the 'draw' method should be implemented in the classes that represent figures.